custom_components/fordpass/sensor.py

```diff
@@ -70,93 +75,109 @@ def get_value(self, ftype):
                 if self.fordoptions[CONF_DISTANCE_UNIT] is not None:
                     if self.fordoptions[CONF_DISTANCE_UNIT] == "mi":
                         if DISTANCE_CONVERSION_DISABLED in self.fordoptions and self.fordoptions[DISTANCE_CONVERSION_DISABLED] is True:
-                            return self.coordinator.data[self.sensor]["value"]
+                            return self.data[self.sensor]["value"]
                         return round(
-                            float(self.coordinator.data[self.sensor]["value"]) / 1.60934
+                            float(self.data[self.sensor]["value"]) / 1.60934
                         )
-                    return self.coordinator.data[self.sensor]["value"]
-                return self.coordinator.data[self.sensor]["value"]
+                    return self.data[self.sensor]["value"]
+                return self.data["odometer"]["value"]
```

Comment:
I have no idea why this fixed odometer showing up in my setup, but I changed all of them and rebooted.
That didn't work so I reverted back, but forgot that one, and then it was working.
Figured I would just throw it in the PR